packages/polymath-ui/src/components/STOStatus/STOStatus.js

```diff
@@ -133,7 +178,7 @@ export default class STOStatus extends Component<Props> {
               </div>
             </div>
           </div>
-          {countdownProps != null && (
+          {countdownProps != null && !isSaleClosed ? (
```

Comment:
`isSaleClosed` doesn't actually get set to `true` anywhere. So it will always be `false` anyway. I think something is missing here.

Reply:
That's correct. The `isSaleClosed` is no longer necessary so I've removed it